error.py

```diff
@@ -36,7 +36,7 @@ class WebsiteConnectionError(StampifierError):
     """Raise when connection to website is not possible"""
 
     def __init__(self, url):
-        super().__init__('Cannot connect to URL! {}'.format(url))
+        super().__init__('Cannot connect to URL!')
```

Comment:
What was the error? Earlier error message was more verbose and probably better.

Reply:
@KomalBhadani27 could you clarify this?

Reply:
Yes it was better but while validating the URL, the original url is converted to **None** if it's not valid or will have **http** prefix appended so technically it won't be the url that user provided. That is why we can't show the url for which error occurred to user. 
Though on web app original url link will show the url that user gave so they can cross check the url from there if needed.

Reply:
Will showing the url still be useful for debugging purposes?

Reply:
Yes, when the provided url pattern is valid and connection error occurred because of some other reason like the url doesn't exist.

Reply:
Where are we using url?

Reply:
The original website url that user provided. The error will be raised if that is invalid or connection cannot be created to that.

Reply:
I think we should remove url. It is added by mistake because of merge conflicts.